src/commands/distribute/lib/distribute-util.ts

```diff
@@ -96,3 +96,19 @@ export async function addGroupToRelease(options: AddGroupToReleaseOptions): Prom
     throw failure(ErrorCodes.Exception, `Could not add ${destinationType} ${destination} to release ${releaseId}`);
   }
 }
+
+export function parseDistributionGroups(groups: string): string[] {
+  debug(`Parsing group string: ${groups}`);
```

Comment:
```suggestion
```

Reply:
Let's remove this one as well. I guess it was added for debug purpose only.

Reply:
done!